controllers/rcj_soccer_referee_supervisor/rcj_soccer_referee_supervisor.py

```diff
@@ -7,11 +7,14 @@
 from referee.referee import RCJSoccerReferee
 from recorder.recorder import VideoRecordAssistant
 
-automatic_mode = False
+from os import environ
 
-TEAM_YELLOW = "The Yellows"
-TEAM_BLUE = "The Blues"
+automatic_mode = True if "RCJ_SIM_AUTO_MODE" in environ.keys() else False
 
+TEAM_YELLOW = environ.get("TEAM_YELLOW_NAME", "The Yellows")
+TEAM_YELLOW_ID = environ.get("TEAM_YELLOW_ID")
+TEAM_BLUE = environ.get("TEAM_BLUE_NAME", "The Blues")
+TEAM_BLUE_ID = environ.get("TEAM_BLUE_ID")
```

Comment:
We should create some kind of validator that in case of automatic mode, we expect the ID, otherwise raise an exception.

Reply:
And we should set some default value because we end up having `None` in the name of json file

Reply:
@Adman or we could perhaps fail completely if this this is not provided?

Reply:
I just put broken code into the supervisor on purpose. It can crash without webots exiting which would clog the tournament simulation pipeline. How complex would it be to exit out of webots if we don't get valid variables?

Reply:
@dbscoach ah, you are right, webots will be running even if supervisor script fails.